numba/tests/test_inlining.py

```diff
@@ -151,6 +152,36 @@ def test_impl(a):
 
         self.assertEqual(len(func_ir._definitions['b']), 2)
 
+    @skip_unsupported
+    def test_inline_var_dict_ret(self):
+        # make sure inline_closure_call returns the variable replacement dict
+        # and it contains the original variable name used in locals
+        @numba.njit(locals={'b': numba.float64})
+        def g(a):
+            b = a+1
+            return b
+
+        def test_impl():
+            return g(1)
+
+        inline_out = None
+        func_ir = compiler.run_frontend(test_impl)
+        blocks = list(func_ir.blocks.values())
+        for block in blocks:
+            for i, stmt in enumerate(block.body):
+                if (isinstance(stmt, ir.Assign)
+                        and isinstance(stmt.value, ir.Expr)
+                        and stmt.value.op == 'call'):
+                    func_def = guard(get_definition, func_ir, stmt.value.func)
+                    if (isinstance(func_def, (ir.Global, ir.FreeVar))
+                            and isinstance(func_def.value, CPUDispatcher)):
+                        py_func = func_def.value.py_func
+                        inline_out = inline_closure_call(
```

Comment:
For clarity, perhaps explicitly unpack this as `_, var_map = inline_closure_call(...` ?

Reply:
I thought checking output as tuple below is better to throw error in assertions, but maybe that's not required. Will change.

Reply:
done.

Reply:
was of the view that the explicit unpacking of the tuple would catch anything accidentally making this tuple larger than 2